integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/passthru/transport/test/SetPropertyMessageBuilderInvokedWithEmptyContentTypeTestCase.java

```diff
@@ -65,22 +60,13 @@ public void testMsgBuilderInvokedPropertyWhenContentTypeisEmpty() throws Excepti
         String apiEp = getApiInvocationURL("setMessageBuilderInvokedWithEmptyContentType");
         jsonclient.sendUserDefineRequest(apiEp, jsonPayload.trim());
 
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
-        Assert.assertTrue(Utils.checkForLog(logViewerClient, "messageBuilderInvokedValue = true", 20));
+        CarbonLogReader carbonLogReader = new CarbonLogReader();
+        carbonLogReader.start();
 
-        LogEvent[] logs = logViewerClient.getAllSystemLogs();
-        for (LogEvent logEvent : logs) {
-            String message = logEvent.getMessage();
-            if (message.contains(EXPECTED_ERROR_MESSAGE)) {
-                isErrorFound = true;
-            }
-        }
-        Assert.assertFalse(isErrorFound);
-    }
+        Assert.assertTrue(carbonLogReader.checkForLog("messageBuilderInvokedValue = true", 20));
+        Assert.assertFalse(carbonLogReader.assertIfLogExists(EXPECTED_ERROR_MESSAGE));
```

Comment:
Shall we add assert messages to the above?

Reply:
Fixed in https://github.com/wso2/micro-integrator/pull/580